hedera-mirror-web3/src/main/java/com/hedera/mirror/web3/evm/account/AccountAccessorImpl.java

```diff
@@ -48,14 +53,29 @@ public boolean isTokenAddress(Address address) {
     }
 
     public Address getAddressOrAlias(final Address address) {
+        // An EIP-1014 address is always canonical
+        if (!aliases.isMirror(address)) {
+            return address;
+        }
+
+        final var account = store.getAccount(address, OnMissing.DONT_THROW);
         final ByteString alias;
-        if (!mirrorEntityAccess.isExtant(address)) {
+
+        if (account == null) {
```

Comment:
I don't think the store can return null. Also think you need to check empty account.
```suggestion
        if (account.isEmptyAccount()) {
```

Reply:
Fixed